targets/lenovo-x1-carbon.nix

```diff
@@ -134,7 +136,7 @@
                   }
                   {
                     name = "zathura";
-                    packages = [pkgs.zathura];
+                    packages = [pkgs.zathura (pkgs.callPackage ../user-apps/demo-pdf {})];
```

Comment:
You could also just put `demo-pdf` here instead of the pkgs.callPackage.

Also Note that you are using pkgs from the host and not from the appvm guest.